src/test/java/nl/atos/ontwikkelstraat/OntwikkelstraatApplicationTest.java

```diff
@@ -0,0 +1,26 @@
+package nl.atos.ontwikkelstraat;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Created by nerom on 31-3-2016.
+ */
+public class OntwikkelstraatApplicationTest {
+
+    @Before
+    public void setUp() {
+
+    }
+
+    @After
+    public void tearDown() {
+
+    }
+
+    @Test
+    public void mainApplication() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/vinsterdamus/Ontwikkelstraat/file/2345586035/issues/source?bid=3138095&fileBranchId=3208141#l23)